### PR TITLE
Added Multiple Templates for Laravel & Symfony Frameworks Misconfigurations

### PR DESCRIPTION
**Pull request by yasinyilmaz**
**Summary:**
Symfony and Laravel based applications, like many similar frameworks it includes a diferrent debug modes with a special interface, allowing developers to view the internal state of network connections for the purpose of identifying errors and misconfigurations, before going production. If debug mode enabled, some systems show the full details of cleartext HTTP request, responses, passwords, tokens and app secrets in debug mode.

**References:**
- https://github.com/itsgoingd/clockwork
- https://github.com/laravel/telescope
- https://symfony.com/doc/current/profiler.html

### CODE REVIEW

File: other/exposed-laravel-clockwork.bcheck [hunk 1]
```diff
@@ -0,0 +1,26 @@
+metadata:
+    language: v1-beta
+    name: "Exposed Laravel Clockwork Endpoint"
+    description: "A Laravel Clockwork php dev tools was discovered, which exposed cleartext HTTP request, responses, passwords, tokens and some information containing app secrets. "
+    author: "Yasin Yilmaz"
+	tags: "laravel", "clockwork", "php"
```
Comment (PortSwiggerWiener) [inline other/exposed-laravel-clockwork.bcheck:6]: Please can you make the indentation consistent.

**Review by PortSwiggerWiener — COMMENTED** [review]
Looks good.

File: other/exposed-laravel-clockwork.bcheck (hunk 1 above)
Comment (yasinyilmaz) [reply other/exposed-laravel-clockwork.bcheck]: Indentation issues fixed.

**Review by PortSwiggerWiener — APPROVED** [review]
👍

**Comment by michael-eaton-portswigger** [comment]
@yasinyilmaz As a contributor to our GitHub repository, we would like to invite you to our closed Discord community.

It is a place where passionate Burp users, including people who directly work on building and developing Burp here at PortSwigger, can talk about the tooling and web security in general.

If you would like to join, please email us at [support@portswigger.net](mailto:support@portswigger.net) and we will send over an invite link.

Thank you!

**Pull request merged**